functions/lib/version.js

```diff
@@ -1,4 +0,0 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
are all these files removed on purpose? What does it have to do with the PR?

Reply:
all the files under `lib` are generated from the compile process.  these files have to be checked in for the extension to work.  you should ignore these files and review all the files under `src`.

Reply:
I see that, but they are all removed, is that expected @smomin ?

Reply:
thank you for catching this.